### ISSUE TITLE
End of project? Status of development

### ISSUE THREAD

**Issue by Neurovert** [MyBBIr/MyBB-CKEditor#73]
@ATofighi did you stop developing this plugin? Please let the community know.

**Comment by ATofighi** [comment]
No, but I don't have free time to develop it for now... I'll try to find free time to fix the issues of the plugin.

I also develop another MyBB-CKEditor plugin for MyBB which has fewer features: https://github.com/ATofighi/mybb-simple-ckeditor

Regards,
AliReza Tofighi Mohammadi
